CONTRIBUTING.md

```diff
@@ -1,11 +1,9 @@
 # Contributing to Docker
 
-Want to hack on Docker? Awesome!  We have a contributor's guide that explains
+Want to hack on Docker? Awesome!  We have a [contributor's guide](https://docs.docker.com/opensource/project/who-written-for/) that explains
 [setting up a Docker development environment and the contribution
 process](https://docs.docker.com/opensource/project/who-written-for/). 
 
-[![Contributors guide](docs/static_files/contributors.png)](https://docs.docker.com/opensource/project/who-written-for/)
```

Comment:
Looks like this image was removed as part of the move to the central docs repository and can now be found here https://github.com/docker/docker.github.io/blob/master/engine/static_files/contributors.png perhaps we should put it back in this repository